Assets/Scripts/PickUp.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class PickUp : MonoBehaviour
+{
+    public Transform Hold;
+
+    void OnMouseDown()
+    {
+        GetComponent<Rigidbody>().useGravity = false;
+        this.transform.position = Hold.position;
+        this.transform.parent = GameObject.Find("HoldingPlace").transform;
```

Comment:
No reason to GameObject.find again. This object you're finding is already saved in the variable Hold.
Also, in another class you named a variable like this _myVariable, but here you named it with capital first letter.
Please try to stick to one naming convention. Preferably this one: _myVariable.

Reply:
There is a reason for it, if i remove it then the picked up object remains in one place instead of moving with the player camera. I tried to change the variables like you said (startLowerCaseThenUpperForEveryWord), I'll try to keep this in mind